woocommerce/woocommerce-paypal-payments
# Axo: Add PayPal Insights integration to the Block Checkout (3770)

### Description

This PR:
- Adds PayPal Insights integration to the Block Checkout (as a standalone Block Editor plugin).
- Adds the `end_checkout` tracking to the `order_received` endpoint.
- Fixes minor bugs in the current PayPal Insights integration.

### Steps to test

1. Visit the Classic Checkout and Block Checkout.
2. Ensure the PayPal Insights events are correctly triggered (verify via the console log).
3. Complete the order and verify the events got correctly triggered on the Order Received page.

### Screenshots

N/A
